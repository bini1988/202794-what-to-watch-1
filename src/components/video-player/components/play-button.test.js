import React from "react";
import renderer from "react-test-renderer";
import PlayButton from "./play-button";

it(`PlayButton correctly renders default markup`, () => {
  const tree = renderer.create(
      <PlayButton onClick={() => {}}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
