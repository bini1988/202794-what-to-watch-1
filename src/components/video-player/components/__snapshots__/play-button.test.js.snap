// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayButton correctly renders default markup 1`] = `
<button
  className="player__play"
  onClick={[Function]}
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</button>
`;
